core/domain/translation_domain.py

```diff
@@ -0,0 +1,109 @@
+# coding: utf-8
+#
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Models for machine translations retrieved from Google Cloud Translate."""
```

Comment:
Add a line above this (I think we do this in other files).

Reply:
Done. I think "Models" is also an inaccurate descriptor here, I changed it to "domain objects".